internal/pkg/model/naming.go

```diff
@@ -30,6 +30,7 @@ type Naming struct {
 	Branch              utils.PathTemplate `json:"branch" validate:"required"`
 	Config              utils.PathTemplate `json:"config" validate:"required"`
 	ConfigRow           utils.PathTemplate `json:"configRow" validate:"required"`
+	Schedule            utils.PathTemplate `json:"schedule" validate:"required"`
```

Comment:
Drzal by som tu rovnaky naming fieldov:
- komponenta sa vola `keboola.variables`, ... field `VariablesConfig`.
- komponenta sa vola `keboola.scheduler`, ... field `SchedulerConfig` 
- to som mohol asi povedat na calle, ale nenapadlo mi to, ... myslim, ze to mas rychlo zmenene, refactor v IDE + regexp v testoch

Reply:
(rovnako aj v JSON `schedule` -> `schedulerConfig`)

Reply:
Aha, ok.